mobile_app_connector/models/frequently_asked_questions.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2019 Compassion CH (http://www.compassion.ch)
+#    @author: Théo Nikles <theo.nikles@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+import logging
+from odoo import models
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import requests
+    from bs4 import BeautifulSoup
+except ImportError:
+    _logger.log('Please install bs4 and requests')
+
+
+class PrivacyStatementAgreement(models.Model):
```

Comment:
Since this class has only one method and no fields, it can be Abstract and therefore won't need any security access rules.
```suggestion
class FAQ(models.AbstractModel):
```